src/test/java/com/squareup/kotlinpoet/CodeBlockTest.kt

```diff
@@ -107,8 +83,8 @@ class CodeBlockTest {
     val map = LinkedHashMap<String, Any>()
     map.put("text", "tacos")
     val block = CodeBlock.builder()
-        .addNamed("%>\n%text:L for %%3.50", map).build()
-    assertThat(block.toString()).isEqualTo("\n    tacos for %3.50")
+        .addNamed("⇥\n%text:L for\n⇤%%3.50", map).build()
```

Comment:
Any specific reason for this test change? Does the code still work with the old test case?

Reply:
Yes, it still works. I changed this test case as a result of encountering that bug I mentioned above. The key difference here is that there is a single no-arg placeholder in the middle of code block.